src/veterinary/VetShop.java

```diff
@@ -1,14 +1,21 @@
 package veterinary;
 import animals.Dog;
-import animals.FurType;
 import animals.Animal;
 import animals.Cat;
 import java.util.ArrayList;
-import java.util.*;
 import java.util.List;
 import java.util.Random;
 
 public class VetShop {
+	/**
```

Comment:
double star not applicable here. Move comments to be above the relevant code

Reply:
fixed to one star

Reply:
moved comments to be above relevant code